src/assets/styles/overrides.scss

```diff
@@ -108,14 +108,21 @@
 }
 
 // Checkboxes
-.v-input--checkbox label {
-  color: $gray7 !important;
+.v-input--checkbox .v-input__control .v-input__slot {
+  label {
+    color: $gray7;
+  }
 }
 
 // Disabled Checkboxes
-.theme--light.v-input--selection-controls.v-input--is-disabled:not(.v-input--indeterminate) .v-icon {
-  color: $app-blue !important;
```

Comment:
I remember an explicit UI requirement to make this app-blue, but the requirement has changed.

Actually, I think the new style looks better so not complaining:

![image](https://user-images.githubusercontent.com/10002889/156081500-674ba997-2b0b-4aab-ab40-8f24ae5b75cb.png)